File: Services/ProcessedNotice.cs

using ScanBus.Models;

namespace ScanBus.Function;
public class ProcessedNotice
{
    public int Id { get; set; }
    public string BusId { get; set; }
    public string Driver { get; set; }
    public Address Address { get; set; }
    public string Latitude { get; set; }
    public string Longitude { get; set; }
    public string LicensePlate { get; set; }
    public string Base64Image { get; set; }
    public string Base64ImageCar { get; set; }
    public string Base64ImageLicensePlate { get; set; }
    public string ImageContentType { get; set; }
}

File: App/src/api/processedNotice.ts
class ProcessedNotice {
    constructor(
        public Id: number,
        public BusId: string,
        public Driver: string,
        public Address: Address,
        public Latitude: string,
        public Longitude: string,
        public LicensePlate: string,
        public UTCTimeStamp: string,
        public ImageContentType: string,
        public Base64Image: string,
        public Base64ImageCar: string,
        public Base64ImageLicensePlate: string
    ) { }
}